.github/actions/build-frontend/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Build Frontend
```

Comment:
❓ Didn't we had this already as a workflow or action before? 🤔 Can we use also in other places then?

Reply:
we have one dedicated to Tasklist https://github.com/camunda/camunda/blob/main/.github/actions/build-tasklist-fe/action.yml
@pedesen is also currently working on using GHA to build operate frontend https://github.com/camunda/camunda/issues/19085. I will get in touch with him to avoid overlapping work